src/redux/books/actions.js

```diff
@@ -0,0 +1,35 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import axios from 'axios';
+
+const APP_ID = 'rMCxybjWy2nJxNQ9XHZB';
+export const getBooks = createAsyncThunk('books/getBooks', async (URL, { rejectWithValue }) => {
+  try {
+    const data = await axios.get(`${URL + APP_ID}/books`);
+    return data.data;
+  } catch (error) {
+    return rejectWithValue(error.message);
+  }
+});
+
+export const addBook = createAsyncThunk('books/addBook', async ({ URL, newBook }, { rejectWithValue }) => {
+  try {
+    const data = await axios.post(`${URL + APP_ID}/books`, newBook, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    return data.data;
+  } catch (error) {
+    return rejectWithValue(error.message);
+  }
+});
```

Comment:
- [ ] No doubt you have expended much effort to pull this but I think there's some bug somewhere that is not allowing the `add book` functionality to work well. Books are being submitted but not getting retrieved.
Here's a [video demo](https://watch.screencastify.com/v/ZQcrJs2GTwEu1dSUu8h2) of the review and discovery, please watch :wink: 

- Kindly check your code to be what this is all about.

Reply:
Hi @AbayomiOlaoye,
I watched the video, It is working fine in my laptop, Maybe there is some problem with your laptop or internet, Can you please check again.